### DISCUSSION TITLE
提问，关于安省（多伦多）college读两年移民

### DISCUSSION THREAD

**Discussion by peforms** [The-Run-Philosophy-Organization/run#142] [General]
目前大学，现在找的移民中介老师准备明年六月毕业去多伦多读那里的college，两年，说读出来干对应专业的工作1-2年可以申请永居或者移民。有大佬评价下靠谱吗，网上搜安省要读研究生什么的才好申请移民，还是说可以多伦多读完college去其他省申请移民，求解＞＜

**Comment by stackunderflow111** [comment]
你本科毕业了去读college属于倒读了，签证很难弄。你应该是去读的研文吧？研文倒是可以移民，但是你要说研文还是master好那肯定master好。总之就是，如果能弄得master，就去读master。不然就去读研文。实在没办法了，去读college，但是签证这方面要多多准备。

**Comment by peforms** [comment]
> 你本科毕业了去读college属于倒读了，签证很难弄。你应该是去读的研文吧？研文倒是可以移民，但是你要说研文还是master好那肯定master好。总之就是，如果能弄得master，就去读master。不然就去读研文。实在没办法了，去读college，但是签证这方面要多多准备。

T T 不是的，现在是专科，九月大三，好像上半学期开始实习，好像可以不去，目前打算一直学习雅思到出国，老师说最好学到5.5，最好六分去那不用读语音强化班了
之前就在想签证的事，他们说🍁签证不好过，一次不过后面都不过；那个中介老师说会帮我写好一点的学习计划，还有准备材料不能说谎要真实，gpa高一些好，这样子比较容易过签证

**Comment by the-run-philosophy** [comment]
转换成讨论

**Comment by Louise0913** [comment]
你好，第一次看到和我经历类似的朋友！我也是一名大专在读生，准备走23fall来着。
语言方面的话，如果你可以坚持学习雅思考到小分6的话，你可以考虑走SDS program！据说这个下学签下得快，但是最近好像有很多声音说普签下的速度比SDS要快了……个人觉得有可能是有普签多人工审核，SDS多AI审核的因素在？（我也不能确定orz）
其他关于润的问题，我也还在研究中……希望你找的移民中介足够可靠！总之避雷所谓大机构！！！
（这是我第一次在github上评论留言，希望没有打扰到你！🥺）

**Reply by peforms** [reply]
你好，很高兴跟你交流，我找的是算是个人中介一样的，途径是朋友的女儿就是他送出去的。我今天去问了中介，说是就走普签，雅思我感觉好难呀，中介说出国前5.5-6最好，没到6还要去那读语言班

**Comment by JJJken404** [comment]
读college的话，普通专业，如工程类，会计之类的，现在要申联邦的移民项目很难，现在EE的分数比较高，只有college diploma的话，除非你雅思考到几乎满分，否则很难过EE的线。建议重点关注以下两点点：
1. 中部几个草原省和海洋三省的移民项目，去的人相对较少，竞争没有那么激烈
2. 关注医护类的移民项目，如护士和护理方向的，这个类别的有自己专门的移民通道，比EE要快和简单许多，而且加拿大也很需要这方面的人，我认识好些人都是走这个移民的
希望可以帮到你

**Reply by peforms** [reply]
感谢你提供的信息！上次我问中介说是b类工作，应该是普通专业，我也不知道好不好申请，她说我年轻分数高啥的，她说送出过好几个比我年龄大的，中专学历的，让我不要担心，其实我现在出去都没出去，还是非常担心了
那几个偏远的省申请是不是要读当地的学校，我读的不是那几个省的学校会不会不好申请
话说申请永居跟入籍是一个东西吗？如果提名不了一直申请工签能混多久呢＞＜我真不想呆在这里了…

**Reply by JJJken404** [reply]
B 类是技术类的专业，我以前就是走的B 类移民的。College好申请的，这个不需要担心。这个网址可以计算EE的分数（不是广告，我也不是这间公司的雇员）https://eoivisa.com/ee/
里面有联邦EE分数的历史数据，你可以根据你的情况去对比一下。疫情期间的分数偏低，但随着生活回归正常，分数也会回归正常。

那几个偏远省你要自己上当地的移民网站看资料，我也不是熟，只知道那边移民会相对简单。下面是那几个省的移民网站的链接，你可以看看。
萨省：
https://www.saskatchewan.ca/residents/moving-to-saskatchewan/live-in-saskatchewan/by-immigrating/saskatchewan-immigrant-nominee-program
Manitoba:
https://immigratemanitoba.com/immigrate-to-manitoba/
Nova Scotia:
https://novascotiaimmigration.com/
New Brunswick:
https://www.welcomenb.ca/content/wel-bien/en.html
Prince Edward Island:
https://www.princeedwardisland.ca/en/topic/office-immigration

永居跟入籍不一样，不过拿到永居就是润成功了。工签一般毕业给三年，找到工作后，如果工签过期了，可以再申一年的桥梁工签。

P.S. 不要完全相信中介，多自己做功课，我就是个给中介坑了的例子，后来我的移民和我老婆的移民我都是自己办的。

**Reply by Louise0913** [reply]
冒昧打扰，实在不好意思！但我还是想再确定一下😢如果目标意在移民，而且无意学习医护类专业的话，就算我在安省读出了cs相关的Diploma学位，是不是移民的机会也不是很大？（在这种情况下尝试fst program和fsw program会有所帮助吗？）在这种假设下，如果我大专毕业后再选择到🍁读研文，移民成功概率会更好吗？

**Reply by JJJken404** [reply]
> 冒昧打扰，实在不好意思！但我还是想再确定一下😢如果目标意在移民，而且无意学习医护类专业的话，就算我在安省读出了cs相关的Diploma学位，是不是移民的机会也不是很大？（在这种情况下尝试fst program和fsw program会有所帮助吗？）在这种假设下，如果我大专毕业后再选择到🍁读研文，移民成功概率会更好吗？

算了下，有难度，如果最高学历是本科的话，需要雅思高分，7到8分，和两年以上的工作经验才能比较稳地申请到。如果能拿到Master的学历，会简单很多。FSW我也不太清楚，身边有两个人申请过，不过他们是多年的海外工作经验加几乎满分的雅思，不具备参考性。
